liquibase-standard/src/main/java/liquibase/changeset/ChangeSetService.java

```diff
@@ -89,6 +89,17 @@ ChangeSet createChangeSet(String id, String author, boolean alwaysRun, boolean r
      */
     ModifyChangeSets createModifyChangeSets(ParsedNode node) throws ParsedNodeException;
 
+    /**
+     *
+     * Create the ModifyChangeSets instance which will do the modifications
+     *
+     * @param  runWith                    The runWith value
+     * @param  runWithSpool               The runWithSpool value
+     * @return ModifyChangeSets           The object which will perform the modifications
+     *
+     */
+    ModifyChangeSets createModifyChangeSets(String runWith, String runWithSpool);
```

Comment:
Can this get a default implementation? I think this breaks backwards compatibility as is, but maybe I'm mistaken.

Reply:
Good point.  I'll add that.